g/github.com__uber__jaeger-client-go/github.com__uber__jaeger-client-go_rhel_8.3.sh

```diff
@@ -21,38 +24,38 @@ PACKAGE_URL=https://github.com/uber/jaeger-client-go
 
 yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
 
-wget https://golang.org/dl/go1.16.1.linux-ppc64le.tar.gz && tar -C /bin -xf go1.16.1.linux-ppc64le.tar.gz && mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
-
+wget https://golang.org/dl/go1.15.linux-ppc64le.tar.gz && tar -C /bin -xf go1.15.linux-ppc64le.tar.gz && mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
```

Comment:
Is there a way to make GOLANG version dynamic?

Reply:
@afsanjar Yes